hypothesis-python/tox.ini

```diff
@@ -42,6 +42,14 @@ deps=
 commands=
     python -m pytest tests/cover/ -n2
 
+[testenv:py36]
```

Comment:
No need to add this section - we already run the py3 tests under Python 3.6 with typing available (and have a fairly complicated build system).

Reply:
sure. removed.